### PR TITLE
SME and QE PR for Bxmsdoc 2432 master - Managing assets in Business Central

### PR DESCRIPTION
**Pull request by michelehaglund**
Please review and test document. Provide your comment either in the Jira or directly in the pull request.

Rendered output: http://file.rdu.redhat.com/~mhaglund/BXMSDOC-2432
Original Jira: https://issues.jboss.org/browse/BXMSDOC-2432

### CODE REVIEW

File: docs/product-user-guide/src/main/asciidoc/assets-types-ref.adoc [hunk 1]
```diff
@@ -1,23 +1,91 @@
...
+Rules provide the logic for the rule engine to execute against.
+A rule includes a name, attributes, a `when` statement on the left hand side of the rule, and a `then` statement on the right hand side of the rule.
+
+* Guided Rule Template
++
+Guided rule templates provide a reusable rule structure for multiple rules that are compiled into Drools Rule Language (DRL) and form the core of the decision service for your project.
+
+* Package
++
+All assets are contained in packages in Drools Workbench. A package is like a folder (it also serves as a "namespace"). A home folder for rule assets to live in.
```
Comment (barboras7) [inline docs/product-user-guide/src/main/asciidoc/assets-types-ref.adoc]: IMHO, Drools Workbench is community naming

File: docs/product-user-guide/src/main/asciidoc/assets-unlocking-proc.adoc [hunk 2]
```diff
@@ -8,17 +8,20 @@ However, you can force unlock an asset if you need to edit a file that is locked
...
 
 . Go to *Menu* -> *Design* -> *Projects* and click the project name.
-. Select the asset from the list to open the asset editor in the *Project Explorer*.
+. Select the asset from the list to open the asset designer in the *Project Explorer*.
 . Go to *Overview -> Metadata* and view the *Lock Status*.
 +
+.Unlock metadata view
+image::unlock-lock.png[Unlock Metadata View]
++
 If the asset is already being edited by another user, the following will be displayed in the *Lock status*
```
Comment (barboras7) [inline docs/product-user-guide/src/main/asciidoc/assets-unlocking-proc.adoc]: There is also a lock icon next to the asset name once the asset is locked.

File: docs/product-assembly_managing-assets/src/main/asciidoc/pam/master-docinfo.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<productname>{PRODUCT}</productname>
+<productnumber>{PRODUCT_VERSION}</productnumber>
+<subtitle>
+</subtitle>
+<abstract>
+	<para>This document explains how to mangage, modify, create, and delete assets {CENTRAL}.</para>
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply docs/product-assembly_managing-assets/src/main/asciidoc/pam/master-docinfo.xml]: Corrected.

File: docs/product-user-guide/src/main/asciidoc/assets-creating-proc.adoc [hunk 4]
```diff
@@ -1,10 +1,10 @@
 [id='creating_assets_proc_{context}']
-= Creating Assets
+= Creating assets
 
 You can create business processes, rules, DRL files, and other assets directly in {CENTRAL} and associate them with the projects that you create.
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply docs/product-user-guide/src/main/asciidoc/assets-creating-proc.adoc]: Good catch. Reworded as follows:
You can create business processes, rules, DRL files, and other assets in your {CENTRAL} projects.

File: docs/product-user-guide/src/main/asciidoc/assets-filtering-proc.adoc [hunk 5]
```diff
@@ -1,37 +1,39 @@
...
-= Filtering Assets by Tag
+= Filtering assets by tags
 
 You can apply tags in the metadata of each asset and then group assets by tags in the Project Explorer. This feature helps you quickly search through assets of a specific category.
 
 .Procedure
 . Go to *Menu* -> *Design* -> *Projects* and click the project name.
-. Select the asset from the list to open the asset editor in the *Project Explorer*.
-. In the asset editor window, go to *Overview -> Metadata*.
+. Select the asset from the list to open the asset designer in the *Project Explorer*.
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply docs/product-user-guide/src/main/asciidoc/assets-filtering-proc.adoc]: Another good question. I struggled with this and referred to https://docs.google.com/spreadsheets/d/1kmK92hY3JJp7gZTvLzd0CawaZzvI2MYYv2uaLaQcMJc/edit#gid=1999150540

Still wasn't confident. I am going to change it to editor for now.

File: docs/product-user-guide/src/main/asciidoc/assets-creating-proc.adoc [hunk 6]
```diff
@@ -1,10 +1,10 @@
...
 
 .Procedure
-. In {CENTRAL}, go to *Menu* -> *Design* -> *Projects* and click the project name. For example, *Evaluation*. (If you have not yet created a project, see xref:projects-con_chap-project[] for details.)
+. In {CENTRAL}, go to *Menu* -> *Design* -> *Projects* and click the project name. For example, *Evaluation*.
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply docs/product-user-guide/src/main/asciidoc/assets-creating-proc.adoc]: Done. Thanks!

File: docs/product-user-guide/src/main/asciidoc/assets-renaming-proc.adoc [hunk 7]
```diff
@@ -1,12 +1,12 @@
 [id='_assets_renaming_proc']
-= Renaming, Copying, or Deleting Assets
+= Renaming, copying, or deleting assets
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply docs/product-user-guide/src/main/asciidoc/assets-renaming-proc.adoc]: Added info and screen capture.

File: docs/product-user-guide/src/main/asciidoc/assets-types-ref.adoc [hunk 8]
```diff
@@ -1,23 +1,91 @@
...
 Anything that can be stored as a version in the {CENTRAL} artifact repository is an asset. This includes rules, packages, business processes, decision tables, fact models, and domain specific languages (DSLs).
 
-The following are descriptions of some of the types of asset in {PRODUCT}.
+The following image shows the available assets in in {PRODUCT} {PRODUCT_VERSION}.
 
-include::assets-rules-gloss.adoc[leveloffset=+1]
+image::asset-list.png[]
 
-include::assets-business-rules-gloss.adoc[leveloffset=+1]
+The following sections describe each asset in {PRODUCT} {PRODUCT_VERSION}.
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply docs/product-user-guide/src/main/asciidoc/assets-types-ref.adoc]: Corrected: The following sections describe each asset type in {PRODUCT} {PRODUCT_VERSION}.

File: docs/product-user-guide/src/main/asciidoc/assets-types-ref.adoc [hunk 9]
```diff
@@ -1,23 +1,91 @@
...
 
-include::assets-business-processes-gloss.adoc[leveloffset=+1]
+* Business Process
++
+Business Processes are flow charts that describe the steps necessary to achieve business goals (see the https://access.redhat.com/documentation/en-US/JBoss_Enterprise_BRMS_Platform/5/html-single/BRMS_Business_Process_Management_Guide/index.html[_{PRODUCT} Business Process Management Guide_] for more details).
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply docs/product-user-guide/src/main/asciidoc/assets-types-ref.adoc]: works for me :) changed as suggested.

File: docs/product-user-guide/src/main/asciidoc/assets-types-ref.adoc [hunk 10]
```diff
@@ -1,23 +1,91 @@
...
 
-include::projects-con.adoc[leveloffset=+1]
+* Business Process (legacy)
++
+Starting in {PRODUCT} {PRODUCT_VERSION}, Stunner is the default process designer. As it's still not fully feature complete, {PRODUCT} {PRODUCT_VERSION} also includes support for the _jBPM process designer_ designer. Thus providing two designers for business processes, the _Business Process_, which relies on _Stunner_, and the _Business Process (legacy)_ one, which relies on the legacy process designer.
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply docs/product-user-guide/src/main/asciidoc/assets-types-ref.adoc]: agreed. Removed.

File: docs/product-user-guide/src/main/asciidoc/assets-types-ref.adoc [hunk 11]
```diff
@@ -1,23 +1,91 @@
...
-include::assets-data-models-gloss.adoc[leveloffset=+1]
+* DRL file
++
+A rule file is typically a file with a .drl extension.
+In a DRL file you can have multiple rules, queries and functions, as well as some resource declarations like imports, globals and attributes that are assigned and used by your rules and queries.
+However, you are also able to spread your rules across multiple rule files (in that case, the extension .rule is suggested, but not required) - spreading rules across files can help with managing large numbers of rules. A DRL file is simply a text file.
+
+* DSL definition
++
+Domain Specific Languages (DSLs) are rule languages that are dedicated to problem domains.
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply docs/product-user-guide/src/main/asciidoc/assets-types-ref.adoc]: Modified as follows:
Domain Specific Languages (DSLs) are a way of creating a rule language that is dedicated to your problem domains. A set of DSL definitions consists of transformations from DSL "sentences" to DRL constructs, which lets you use of all the underlying rule language and engine features.

File: docs/product-user-guide/src/main/asciidoc/assets-types-ref.adoc [hunk 12]
```diff
@@ -1,23 +1,91 @@
...
+
+* Enumeration
++
+Data enumerations are an optional asset type that can be configured to provide drop down lists for the guided designer. They are stored and edited just like any other asset, and apply to the package that they belong to.
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply docs/product-user-guide/src/main/asciidoc/assets-types-ref.adoc]: My bad. Correction made. Thank you.

File: docs/product-user-guide/src/main/asciidoc/assets-types-ref.adoc (hunk 1 above)
Comment (michelehaglund) [reply docs/product-user-guide/src/main/asciidoc/assets-types-ref.adoc]: Changed to:

All assets are contained in packages in {CENTRAL}. A package is a folder for rules and also serves as a "namespace".

File: docs/product-user-guide/src/main/asciidoc/assets-unlocking-proc.adoc (hunk 2 above)
Comment (michelehaglund) [reply docs/product-user-guide/src/main/asciidoc/assets-unlocking-proc.adoc]: Added info.

**Pull request merged**